composer.json

```diff
@@ -23,9 +23,9 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4 || ^8.2",
```

Comment:
^7.4 || ^8.0 to be fully compatible with Ibexa 4.6 ? 

https://github.com/ibexa/oss/blob/4.6/composer.json